v2/dtos/requests/operation.go

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package requests
+
+import (
+	"encoding/json"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/common"
+)
+
+// OperationRequest defines the Request Content for SMA POST Operation.
+// This object and its properties correspond to the OperationRequest object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/system-agent/2.x#/OperationRequest
+type OperationRequest struct {
+	common.BaseRequest `json:",inline"`
+	ServiceName        string `json:"serviceName" validate:"required"`
+	Action             string `json:"action" validate:"required"`
```

Comment:
Add the action enum to constants.go, and use `oneof` validation
https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/v2/constants.go

For example, https://github.com/edgexfoundry/go-mod-core-contracts/blob/c99d9b30f8ab6b04b1c765349bd436aa7698dd28/v2/dtos/address.go#L15